Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.11 as build
+
+WORKDIR /
+COPY . /
+ENV CGO_ENABLED 0
+RUN go build -tags 'static' -ldflags '-extldflags "-static"' .
```

Comment:
I would like to (re)check what flags are actually nice to use (and what there use actually means). Here I have added a lot of flags https://github.com/Ilyes512/kubia/blob/master/Dockerfile#L16 Adding `-s -w` removes some debugging things that would reduce the image size (not that it really matters).